### PR TITLE
Fix phpunit tests for MDL-64492

### PR DESCRIPTION
**Pull request by rhell4**
MDL-64492 changed the activity icon output so unit tests now fail

### CODE REVIEW

**Comment by gjb2048** [comment]
Hi Rossco,

This won't work as the version number check has the value of '2018051705.05' which is Moodle 3.5 branch version and thus M3.6 versions before MDL-64492 will now start to fail!

G

File: tests/courseformatrenderer_test.php [hunk 1]
```diff
@@ -260,6 +260,10 @@ public function test_section_hidden() {
 
     public function test_print_multiple_section_page_horizontal() {
         global $CFG;
+        $activityicon = 'alt=" " role="presentation" ';
+        if ($CFG->version >= 2018051705.05) {
```
Comment (gjb2048) [inline tests/courseformatrenderer_test.php]: This is a M3.5 branch version and not a M3.6 branch version!

File: tests/courseformatrenderer_test.php [hunk 2]
```diff
@@ -290,6 +294,10 @@ public function test_print_multiple_section_page_horizontal() {
 
     public function test_print_multiple_section_page_vertical() {
         global $CFG;
+        $activityicon = 'alt=" " role="presentation" ';
+        if ($CFG->version >= 2018051705.05) {
```
Comment (gjb2048) [inline tests/courseformatrenderer_test.php]: This is a M3.5 branch version and not a M3.6 branch version!

File: tests/courseformatrenderer_test.php [hunk 3]
```diff
@@ -320,6 +328,10 @@ public function test_print_multiple_section_page_vertical() {
 
     public function test_print_multiple_section_page_no_sections_horizontal() {
         global $CFG;
+        $activityicon = 'alt=" " role="presentation" ';
+        if ($CFG->version >= 2018051705.05) {
```
Comment (gjb2048) [inline tests/courseformatrenderer_test.php]: This is a M3.5 branch version and not a M3.6 branch version!

File: tests/courseformatrenderer_test.php [hunk 4]
```diff
@@ -341,6 +353,10 @@ public function test_print_multiple_section_page_no_sections_horizontal() {
 
     public function test_print_multiple_section_page_no_sections_vertical() {
         global $CFG;
+        $activityicon = 'alt=" " role="presentation" ';
+        if ($CFG->version >= 2018051705.05) {
```
Comment (gjb2048) [inline tests/courseformatrenderer_test.php]: This is a M3.5 branch version and not a M3.6 branch version!

**Comment by rhell4** [comment]
Hi Gareth,
I've just changed the version to 2018120303.06 which should be the correct version for M3.6

**Comment by gjb2048** [comment]
Thanks Rossco, please could you combine this into one commit to clean things up.  I follow the same practices as MDL's.

**Comment by rhell4** [comment]
Hi Gareth, this should be combined into a single commit now.

**Comment by gjb2048** [comment]
Thanks Rossco.

**Pull request merged**